src/test/kotlin/org/rustPerformanceTests/RsRealProjectTestBase.kt

```diff
@@ -92,13 +92,19 @@ abstract class RsRealProjectTestBase : RsWithToolchainTestBase() {
     }
 }
 
-fun VirtualFile.findDescendants(filter: (VirtualFile) -> Boolean): ArrayList<VirtualFile> {
+fun VirtualFile.findDescendants(exclude: List<String>, filter: (VirtualFile) -> Boolean): ArrayList<VirtualFile> {
     val result = ArrayList<VirtualFile>()
     VfsUtilCore.visitChildrenRecursively(this,
         object : VirtualFileVisitor<ArrayList<VirtualFile>>() {
             override fun visitFile(file: VirtualFile): Boolean {
-                if (!file.isDirectory && filter(file)) result.add(file)
-                return true
+                return if (file.isDirectory) {
+                    file.name !in exclude
```

Comment:
Currently `exclude`  matches relative path prefix (e.g. `src/llvm`) but not a directory name on any level. Also, I don't like changing `findDescendants` to provide such kind of filtering. You can just use `filter` to match a prefix

Reply:
The corresponding change was introduced to filter out `core_arch` module from stdlib. But after #6526 we don't want to do it so I just dropped the change